src/js/components/Select/SelectContainer.js

```diff
@@ -342,6 +342,13 @@ const SelectContainer = forwardRef(
                 type="search"
                 value={search || ''}
                 placeholder={searchPlaceholder}
+                onFocus={() => {
+                  if (search) {
+                    setSearch(search);
+                    setActiveIndex(-1);
+                    onSearch(search);
+                  }
+                }}
```

Comment:
This code is also used on the onChange, since they are the same behaviour maybe we could have a function that holds it, something like:

```js
const executeSearch = useCallback( search =>
  setSearch(search);
  setActiveIndex(-1);
  onSearch(search)
  ,[ hook dependencies here ]
  )
  ...
onFocus={() => search && executeSearch(search)}
onChange={event => executeSearch(event.target.value)}
```